packages/@ember/-internals/runtime/lib/mixins/container_proxy.d.ts

```diff
@@ -1,15 +1,17 @@
-import { Container } from '@ember/-internals/container';
+import Container, { FactoryManager } from '@ember/-internals/container/lib/container';
 import { TypeOptions } from '@ember/-internals/container/lib/registry';
 import { Mixin } from '@ember/-internals/metal';
 import { Factory } from '@ember/-internals/owner';
 
-interface ContainerProxyMixin {
+export interface IContainer {
```

Comment:
IIRC it was to disambiguate.

Reply:
`IContainer` and `IRegistry` are used as part of the `Owner` interface. I don't want to merge with the mixins since `ContainerProxyMixin` since that has the `__container__` property and `RegistryProxyMixin` has the `__registry__` property, neither of which belong on `Owner`.